delivery_package_number/models/stock_picking.py

```diff
@@ -11,9 +12,12 @@ class StockPicking(models.Model):
         compute="_compute_number_of_packages",
         readonly=False,
         store=True,
+        default=1,
```

Comment:
@pedrobaeza Would you want to say there is always a package (which is not the case...)?